docker/kubernetes-tentacle/scripts/configure-and-run.sh

```diff
@@ -327,6 +327,11 @@ function registerAdditionalServer() {
   tentacle "${ARGS[@]}"
 }
 
+function markAsInitialised() {
+    # There is a startupProbe which checks for this file
+    touch /scripts/initialised
```

Comment:
not sure about this directory, but the file idea is 👍 

maybe `/etc/octopus/initialized`?

Reply:
Yeah, initially I just had it as `touch initialised` which goes into `/scripts/` because that's where it's running, then I thought I should be explicit but yeah `/etc/octopus/` sounds better.